jobbergate-api/dev_tools/entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
+set -e
+cd /app
 poetry run dev-tools db upgrade
-poetry run dev-tools dev-server --port=8000
-
-tail -f /dev/null
+poetry run dev-tools dev-server --port=80
```

Comment:
does it mean port 80 will be published in the host level?

Reply:
No. This entrypoint is used for containers, so their external ports will need to be forwarded to port 80 by the orchestrator.